apps/eth-wrapper/public/logo.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```

Comment:
We should remove the React icons and point everything that still used these to the eth wrapper icon.